public/javascript/main.js

```diff
@@ -377,4 +377,16 @@ $(function(){
           $(this).data('chart', chart);
       }
   })
+
+  // Event tracking
+  // Clicks on elements with data-ga-track-click attribute will be recorded
+  function trackClick(eventDescription){
+    eventDescription = eventDescription || 'Undescribed event.';
+    ga('send', 'event', 'user interaction', eventDescription, document.title);
```

Comment:
Removed tracking from events that lead off page. Will implement later today or tomorrow.